### PR TITLE
feat(issue-21): implementation of event card

### PR DESCRIPTION
**Pull request by yoki1234**
Developed as: 

image

### CODE REVIEW

File: src/components/eventTitle/EventTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+
+enum Variation {
```
Comment (Macdeal) [inline src/components/eventTitle/EventTitle.tsx:3]: export enum

File: src/container/EventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <div className="w-full h-full p-4 lg-p-8 rounded-lg border border-gray-800 bg-gray-900">
+      <div className="bg-gray-800 h-48 flex items-center justify-center rounded mb-6 p-4">
+        <img
+          className="w-32 max-h-24"
+          src={imageURL}
+          alt="event"
+        />
+      </div>
+      <div className="flex flex-col items-start">
+        <EventTitle text={title} variation="card-title"/>
```
Comment (Macdeal) [inline src/container/EventCard.tsx:25]: use enum type for variation

**Review by Macdeal — CHANGES_REQUESTED** [review]

File: src/components/eventTitle/EventTitle.tsx (hunk 1 above)
Comment (yoki1234) [reply src/components/eventTitle/EventTitle.tsx:3]: done as required.

File: src/container/EventCard.tsx (hunk 2 above)
Comment (yoki1234) [reply src/container/EventCard.tsx:25]: addressed with the previous review comment.

**Review by Macdeal — APPROVED** [review]

**Pull request merged**